index.js

```diff
@@ -48,8 +48,6 @@ const kAddHookOriginal = Symbol('fastify otel addhook original')
 const kSetNotFoundOriginal = Symbol('fastify otel setnotfound original')
 
 class FastifyOtelInstrumentation extends InstrumentationBase {
-  static FastifyOtelInstrumentation = FastifyOtelInstrumentation
-  static default = FastifyOtelInstrumentation
```

Comment:
Simulating a default export in cjs only works in bundlers. This is node.js code so we need to assign to module.exports.